### PR TITLE
create admin folder

### PR DESCRIPTION
**Pull request by fvankrieken**
Bunch of little things
- `docker` and `python` folders moved into `admin/run_environment`
- `python` folder removed (reqs and constraints moved up into `run_environment`
- `docker` publish scripts tweaked to run from root of project
- `ops` folder created within `admin` - scripts that are run for specific operations. Some of these are run frequently - compile python requirements, clean build artifacts, etc. Some won't be -> create/setup dev bucket

Some of the commits are useful, mainly for specific code changes, but more than anything I would just look at the repo checked out to this branch and see what you think.

[docker publish job working](https://github.com/NYCPlanning/data-engineering/actions/runs/10947559102)
[compile python packages job working](https://github.com/NYCPlanning/data-engineering/actions/runs/10947640879) (and [pr created](https://github.com/NYCPlanning/data-engineering/pull/1147))
[clean build artifacts job working](https://github.com/NYCPlanning/data-engineering/actions/runs/10947561469)
[qa deploy working](https://github.com/NYCPlanning/data-engineering/actions/runs/10947654128/job/30396872340)

### CODE REVIEW

**Comment by sf-dcp** [comment]
may I suggest a small readme file or updated wiki page to have a short description of ops and run environment folders?

File: bash/docker_container_setup.sh
Comment (sf-dcp) [inline bash/docker_container_setup.sh:1]: Do we want to keep it in `bash`? Sorry if I missed this being discussed during the meeting

Comment (fvankrieken) [reply bash/docker_container_setup.sh:1]: I don't think we decided on the couple things left in this folder. They're all sort of "build utils" or something along those lines - this one maybe a little more in-between given that it's a final setup that runs before a build.

I think we just leave them for now

**Comment by fvankrieken** [comment]
> may I suggest a small readme file or updated wiki page to have a short description of ops and run environment folders?

If this gets merged I'll update https://github.com/NYCPlanning/data-engineering/wiki/Repository-Structure

**Review by alexrichey — APPROVED** [review]
Feels like a definite improvement!

**Review by damonmcc — APPROVED** [review]

**Comment by fvankrieken** [comment]
Updated both [repository structure](https://github.com/NYCPlanning/data-engineering/wiki/Repository-Structure) and [docker image management ](https://github.com/NYCPlanning/data-engineering/wiki/Docker-Image-Management)wiki pages

**Comment by sf-dcp** [comment]
ty!

**Pull request merged**